### ISSUE TITLE
Adding a comment does not refresh Reference View/Comments

### ISSUE THREAD

**Issue by nekopep** [x64dbg/x64dbg#999]
When we add a comment in disassembly view (double click right section) the comment view in reference widget is not updated.
User need to press Alt+Ctrl+C to force a refresh or the newly created comment are not shown.

**Comment by mrexodia** [comment]
Yes, that's the expected behavior. The view is not interactive in any way at the moment.
